augly/image/utils.py

```diff
@@ -279,3 +280,23 @@ def compute_stripe_mask(
     binary_mask = softmax_mask > (math.cos(math.pi * line_width) + 1) / 2
 
     return binary_mask
+
+
+def distort(
+        image: Image.Image,
+        distortion_type: str,
+        distortion_args: Tuple,
+) -> Image.Image:
+    """
+    Distorts the image with a specified type of distortion. This function wraps `Wand` package
+    distort() method to apply distortions. This function is an helper function to apply lens distortions
```

Comment:
```suggestion
    distort() function to apply distortions. This function is a helper function to apply lens distortions
```

Reply:
It is actually a classmethod,  though I would change it as `wand.image.Image.distort().